PyFlyt/core/aviary.py

```diff
@@ -107,17 +109,21 @@ def reset(self, seed: None | int = None):
                     start_orn=start_orn,
                     ctrl_hz=self.ctrl_hz,
                     physics_hz=self.physics_hz,
-                    drone_model=self.drone_model,
```

Comment:
Seems like this is not needed here, for both quadx and FW. Since in each `quadx.py` and `fixedwing.py` it has its own drone model name

Reply:
Good catch